src/ButtonAnchor.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+export default class ButtonAnchor extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+  render() {
+    let { href, role, tabIndex, disabled, style } = this.props;
+
+    if (disabled) {
+      tabIndex = -1;
+      style = { pointerEvents: 'none', ...style };
+    }
+
+    return (
+      <a
+        {...this.props}
+        style={style}
+        disabled={null}
+        // we want to make sure there is a href attribute on the node
+        // otherwise, the cursor incorrectly styled (except with role='button')
+        href={href || ''}
+        role={role || 'button'}
+        // http://getbootstrap.com/javascript/#dismiss-on-next-click
+        tabIndex={tabIndex || 0}
+        onClick={this.handleClick}
+      />
+    );
+  }
+
+  handleClick(event) {
+    let { disabled, onClick } = this.props;
+
+    event.preventDefault();
```

Comment:
bah yeah...so there are two cases to address:

1. links that are really buttons
2. real links that just look like buttons

this addresses 1. but kills 2...

I originally had this in SafeAnchor but Button can't use SafeAnchor...or maybe it can. I'll take another look